packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -76,6 +76,10 @@ BuildRequires: (npm(react-bootstrap) >= 0.32.1 with npm(react-bootstrap) < 1.0.0
 BuildRequires: (npm(use-deep-compare-effect) >= 1.6.1 with npm(use-deep-compare-effect) < 2.0.0)
 # end package.json dependencies BuildRequires
 
+# start specfile assets BuildRequires
```

Comment:
This would point to an `assets` bundler group like Foreman has (https://github.com/theforeman/foreman/blob/207137e04d1c87757477d758c20d0790de142173/bundler.d/assets.rb#L1), but katello doesn't use that. Instead, in https://github.com/Katello/katello/pull/10982 you add it to the gemspec so it belongs in the `specfile generated dependencies` group.

Reply:
Thanks, Moved